pandas/core/sorting.py

```diff
@@ -466,18 +467,17 @@ def nargminmax(values: ExtensionArray, method: str, axis: int = 0):
     func = np.argmax if method == "argmax" else np.argmin
 
     mask = np.asarray(isna(values))
-    arr_values = values._values_for_argsort()
```

Comment:
I don't think we should make this change (or otherwise we should pass a mask here as well). Currently, because of passing the EA itself and not an ndarray, we can calculate the man more efficiently/correctly.

Reply:
Hi @jorisvandenbossche, thanks for your input! If I were to pass a mask, let me just see if I got it right: we should calculate the mask in the functions that call nargminmax and pass them on as a parameter to this method, correct? Would that be a better solution?